### ISSUE TITLE
Extension is broken

### ISSUE THREAD

**Issue by drveresh** [meetDeveloper/googleDictionaryAPI#38]
The chrome extension isn't working, always showing "Please Wait...".

**Comment by meetDeveloper** [comment]
@drveresh I just checked it is working in my system, can you uninstall and try again, on which system are you?

**Comment by drveresh** [comment]
@meetDeveloper, I tried it on my Mac Pro with latest Chrome version 75.0.  Let me try again and confirm.

**Comment by meetDeveloper** [comment]
@drveresh Did you tried again, uninstalling and reinstalling it?

**Comment by drveresh** [comment]
@meetDeveloper, yeah I tried, but the same issue, see the screenshot - http://prntscr.com/o5mvdz.  The first request is fine (200), but no response and the next subsequent requests are either 200 or redirecting to another Google URL, see this http://prntscr.com/o5my3f.

Let me know if any fix around this issue.

Also, can I see its Chrome extension code? If so I can help trouble the issue.

**Comment by drveresh** [comment]
@meetDeveloper, I guess, this is also an issue when no response, see - http://prntscr.com/o5n1gj.

**Comment by meetDeveloper** [comment]
@drveresh You can go to installed extension folder, https://stackoverflow.com/questions/17377337/where-to-find-extensions-installed-folder-for-google-chrome-on-mac, and see the code, I will try to correct this today or by tomorrow.

**Comment by meetDeveloper** [comment]
@arjunsingh-pm Yo!

**Comment by meetDeveloper** [comment]
Can you open the issue on https://github.com/meetDeveloper/Dictionary-Anywhere, as this project is for Dictionary API. I will fix the dictionary issue in upcoming weekend.

**Comment by meetDeveloper** [comment]
@drveresh I have fixed the issue, can you confirm once and tell me. https://chrome.google.com/webstore/detail/dictionary-anywhere/hcepmnlphdfefjddkgkblcjkbpbpemac/related?hl=en Sorry it took me lot of time, life was being tough. Kindly check once if it is working. I am really sorry.

**Comment by drveresh** [comment]
Sorry for late reply, it is working perfectly fine.  If possible, can you
also add *Synonyms* and 1-2 usage statements as well?

Thank you.
